runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -291,6 +292,9 @@ public final class AnimationID
 	public static final int CHURN_MILK_LONG = 2795;
 	public static final int CLEANING_SPECIMENS_1 = 6217;
 	public static final int CLEANING_SPECIMENS_2 = 6459;
+	public static final int SACRIFICE_BLESSED_BONE_SHARDS = 11002;
```

Comment:
```suggestion
	public static final int SACRIFICE_BLESSED_BONE_SHARDS = 11102;
```

in my testing

Reply:
I was getting IDs secondhand from someone via Discord since I don't have access to this content currently. Seems like they gave me the incorrect ID for this one.